ts/Series/AreaSeries.ts

```diff
@@ -480,7 +480,7 @@ seriesType<Highcharts.AreaSeries>(
 
                 isNull = points[i].isNull;
                 plotX = pick(points[i].rectPlotX, points[i].plotX);
-                yBottom = pick(points[i].yBottom, translatedThreshold);
+                yBottom = stacking ? points[i].yBottom : translatedThreshold;
```

Comment:
Just a question: is `yBottom` always defined when stacking is set? Or can be e.g. null? On the left we used `pick()`. Should we do the same here? `stacking ? pick(points[i].yBottom, translatedThreshold) : translatedThreshold;`

Reply:
I checked the code and find this part: https://github.com/highcharts/highcharts/blob/master/ts/Core/Series/Series.ts#L5286. Correct me if I am wrong, but it seems that `yBottom` is always defined when stacking is set.

Reply:
Looks good, thanks for checking! 👍